### ISSUE TITLE
Opening .d (Agilent) files on MQ 2.5.2.0

### ISSUE THREAD

**Issue by jcwaddington** [cox-labs/CoxLab_Bug_Reporting#75]
**Describe the bug**
When loading folder, and navigating to .d (Agilent) files, the following error message is produced. 

![image](https://github.com/cox-labs/CoxLab_Bug_Reporting/assets/101057676/c4fc326f-3736-446c-ae0f-f1caf57242dc)

This has been replicated on two different laptops. Both required an updated .NET framework to be installed for MaxQuant to open

**To Reproduce**
1. Go to Load Folder and click Browse
2. Click on Folder containing .d files
3. Click Select Folder

**Expected behavior**
.d files loaded into MaxQuant, as in previous versions

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise
 - Version 22H2 19045.4291

**Comment by jcwaddington** [comment]
Note - downloaded MaxQuant_v_2.4.9.0 and can open .d files no problem

**Comment by yeroslaviz** [comment]
Hi,
Would you be willing to share the raw file? 
Can you upload it [here](https://datashare.biochem.mpg.de/s/oiWZ2jhwCx64alu).

**Comment by jcwaddington** [comment]
Hello,

I've uploaded a blank (zipped), but let me know if you need anything else. I have not been able to get MaxQuant to work on either laptop... it keeps closing without any warning messages about 30 seconds after I start the run. I've used the software a lot in the past on a different PC (which I no longer have access to). Wondering if this is something to do with the .NET frameworks installed?

Thansk,
James

________________________________

**Comment by yeroslaviz** [comment]
Hi James, sorry to come back so late.  

Does the problem still exits? 
Did you try a newer version of MQ since then?

**Comment by jcwaddington** [comment]
Hello, 

I've just downloaded 2.6.7.0 and the issue still exists. Unable to load .d files using the 'Select folder' option, but can still perform searches using 2.4.9.0.

**Comment by yeroslaviz** [comment]
Hi James, sorry for the delay. 


I don't really understand the trouble. It works fine on my side.  I think the problem is you go one level too deep. I have created a short screenCapture [here](https://datashare.biochem.mpg.de/s/hapzLkGRQd2CRnt) for you to understand what I'm doing.

The folder you want to take is the one containing all the folders *.d. 

Let's say you have `1.d`, `2.d` and `3.d` inside the folder `raw_data`.

1. Go to Load Folder and click Browse
2. select folder `raw_data/`
3. Click Select Folder
=> no error :-) 

let me know if it works

Assa

**Comment by yeroslaviz** [comment]
What error do you get?

**Comment by yeroslaviz** [comment]
Is this what you get? 

<img width="961" alt="Image" src="https://github.com/user-attachments/assets/0f55b5b7-d42c-4563-b3a9-15aa3e118946" />

I lieterally created three empty files and tried to upload them. If so, there is a problem with your raw files.

**Comment by jcwaddington** [comment]
The same raw file that can be uploaded v2.4.9.0 cannot be opened in v2.5.2.0.

![Image](https://github.com/user-attachments/assets/c90650c6-9e91-41d1-9d29-c744bd077ef6)

**Comment by yeroslaviz** [comment]
Are these Agilent MassHunter  raw files? 

Unfortunately, at the moment MaxQuant is not equipped with analyzing these files.

**Comment by jcwaddington** [comment]
Yes, these are Agilent .d files. Previous versions of MaxQuant would run and search Agilent .d files without issue. Has this functionality now been dropped?

**Comment by yeroslaviz** [comment]
From what I can see it seems that the Agilent plugin was removed, when we moved to .NET 7.

**Comment by yeroslaviz** [comment]
Hi James, we are putting it back, but we need to test it. Is there a way to get one or two raw files for Agilent? not empty ones. Can you give me a couple or point us to a PRIDE repository?
